config/locales/en.yml

```diff
@@ -60,7 +60,6 @@ en:
   mismatch_declaration_type_for_schedule: "The property '#/declaration_type' does not exist for this schedule."
   missing_declaration_type: "Enter a '#/declaration_type'."
   missing_evidence_held: "Enter a '#/evidence_held' value for this participant."
-  missing_has_passed: "Enter 'true' or 'false' in the '#/has_passed' field to indicate whether this participant has passed or failed their course."
```

Comment:
I think we can revert those since we don't need to touch the record declaration service